polynote-kernel/src/main/scala/polynote/kernel/Result.scala

```diff
@@ -83,7 +83,7 @@ object RuntimeError extends ResultCompanion[RuntimeError](2) {
   )
 
   // extending only to access the protected constructor (which skips building a pointless stack trace)
-  final case class RecoveredException(msg: String, originalType: String) extends RuntimeException(s"$originalType: $msg", null, false, false)
+  final case class RecoveredException(msg: String, originalType: String) extends RuntimeException(s"$originalType: $msg", null, false, true)
```

Comment:
@jeremyrsmith Apparently without this change `setStackTrace` wasn't working - but per the comment it looks like the point of this class was to set this value?